ICAFIX/ReApplyFixMultiRunPipeline.sh

```diff
@@ -36,26 +34,19 @@ set -e # If any command exits with non-zero value, this script exits
 usage()
 {
 	cat <<EOF
-
```

Comment:
Why did you remove these blank lines in the usage info?  Please put them back in.

Reply:
I did not even know they were taken out. I had to re-merge as the code for FIX in my branch and master was not in sync for some reason.